### PR TITLE
Implemented logger

### PR DESCRIPTION
**Pull request by dfordivam**
Details in the file Logger.hs

### CODE REVIEW

File: src/Main.hs [hunk 1]
```diff
@@ -32,16 +33,29 @@ parse _ = exit
...
+    let logMsg = logMessage logR
+    logMsg $ "Parsing input file: " ++ concat args
     torrentStr <- parse args
     case decode torrentStr of
       Right d ->
           case mkMetaInfo d of
-            Nothing -> putStrLn "parse error"
+            Nothing -> logMsg "parse error"
             Just m -> do
+              logMsg "Input File OK"
```
Comment (dfordivam) [inline src/Main.hs:45]: I found an interesting thing, if the "logMsg Input File OK" is removed and there are issues in network connectivity, then the next message "Trying to fetch peers" is not printed immediately (there is delay in the printing of message)

Comment (jaseemabid) [inline src/Main.hs:45]: Should be shell buffering i guess? Can you try again with `hSetBuffering stdout NoBuffering` please?

**Comment by jaseemabid** [comment]
@vu3rdd LGTM.

**Comment by vu3rdd** [comment]
@dfordivam @jaseemabid Sorry, haven't found time to look at it. Will do today. BTW, should we create a org for our haskell bangalore group and move the project there? I don't want to be the bottle neck in the development of the project here.

**Comment by jaseemabid** [comment]
@vu3rdd give commit access to collaborators. Collaborators rather than commit directly to mainline can always raise pull requests (unless trivial changes), get it reviewed by one more person and merge to master.

**Comment by vu3rdd** [comment]
@jaseemabid Good idea! I already gave you commit access, adding @dfordivam as well.

**Comment by dfordivam** [comment]
Ok

It would be great if we also create some milestones for the project, and
then open issues to work on them.

I can think of following milestones
1. Client -tracker communication
2. Peer communication
3. Data integrity checker
4. User Interface / Control
5. Disk IO
6. concurrency

**Pull request merged**